README.md

```diff
@@ -1,66 +1,75 @@
+## Prerequisites
 
-* a working Python 2.7 development environment ([getting started guide](http://hackercodex.com/guide/python-development-environment-on-mac-osx/))
-* experience [deploying Python apps to Heroku](https://devcenter.heroku.com/articles/getting-started-with-python#introduction)
-* [pip](https://pip.pypa.io/en/latest/installing.html) to install Python dependencies
+- a working Python 3.8 development environment ([getting started guide](http://hackercodex.com/guide/python-development-environment-on-mac-osx/))
+- [pip](https://pip.pypa.io/en/latest/installing.html) to install Python dependencies
+- experience [deploying Python apps to Heroku](https://devcenter.heroku.com/articles/getting-started-with-python#introduction)
 
 ## Running the Snake Locally
 
-1) [Fork this repo](https://github.com/battlesnakeio/starter-snake-python/fork).
+1. [Fork this repo](https://github.com/BattlesnakeOfficial/starter-snake-python/fork).
 
-2) Clone repo to your development environment:
-```
-git clone git@github.com:<your github username>/starter-snake-python.git
-```
+2. Clone repo to your development environment:
 
-3) Install dependencies using [pip](https://pip.pypa.io/en/latest/installing.html):
-```
-pip install -r requirements.txt
-```
+    ```shell
+    git clone git@github.com:<your github username>/starter-snake-python.git
+    ```
 
-4) Run local server:
-```
-python app/main.py
-```
+3. Install dependencies using [pip](https://pip.pypa.io/en/latest/installing.html):
 
-5) Test your snake by sending a curl to the running snake
-```
-curl -XPOST -H 'Content-Type: application/json' -d '{ "hello": "world"}' http://localhost:8080/start
-```
+    ```shell
+    pip install -r requirements.txt
+    ```
+
+4. Run local server:
+
+    ```shell
+    python app/main.py
+    ```
+
+5. Test your snake by sending a curl to the running snake
+
+    ```shell
+    curl -XPOST -H 'Content-Type: application/json' -d '{ "hello": "world"}' http://localhost:8080/start
+    ```
 
 ## Deploying to Heroku
 
-1) Create a new Heroku app:
-```
-heroku create [APP_NAME]
-```
+1. Create a new Heroku app:
+
+    ```shell
+    heroku create [APP_NAME]
+    ```
 
-2) Deploy code to Heroku servers:
-```
-git push heroku master
-```
+2. Deploy code to Heroku servers:
+
+    ```shell
+    git push heroku master
+    ```
+
+3. Open Heroku app in browser:
+
+    ```shell
+    heroku open
+    ```
 
-3) Open Heroku app in browser:
-```
-heroku open
-```
 or visit [http://APP_NAME.herokuapp.com](http://APP_NAME.herokuapp.com).
 
-4) View server logs with the `heroku logs` command:
-```
-heroku logs --tail
-```
+4. View server logs with the `heroku logs` command:
+
+    ```shell
+    heroku logs --tail
+    ```
 
 ## Questions?
 
-Email [hello@battlesnake.com](mailto:hello@battlesnake.com), or tweet [@battlesnakeio](http://twitter.com/battlesnakeio).
+Email [hello@battlesnake.com](mailto:hello@battlesnake.com), or tweet [@battlesnakeio](http://twitter.com/BattlesnakeOfficial).
```

Comment:
Still using the old twitter handle.

Reply:
I literally _just_ realized this because of an entirely different reason. Will fix!